lib/PDF/API2/Annotation.pm

```diff
@@ -158,12 +158,14 @@ sub launch {
 
 =head3 pdf
 
-    $annotation = $annotation->pdf($file, $page_number, $location, @args);
+    $annotation = $annotation->pdf($file, $dest, $location, @args);
 
-Open the PDF file located at C<$file> to the specified page number.
+Open the PDF file located at C<$file> to the specified destination.
 C<$location> and C<@args> are optional and set which part of the page should be
 displayed, as defined in L<PDF::API2::NamedDestination/"destination">.
 
+C<$dest> can be a page number, or a named destination prefixed with C<#>.
```

Comment:
Would it make sense to have the "name" vs. "page number" disambiguator be a "/" instead of a "#"?  That would follow the syntax for name objects (see 7.3.5).

Reply:
In a way, yes. On the other hand, people are familiar with `#xxx` since it is widely used in HTML and URL.

(7.3.5 is Knockout Groups in the PDF 1.4 and 1.7 specs???)